torchgeo/trainers/utils.py

```diff
@@ -37,6 +37,9 @@ def extract_backbone(path: str) -> tuple[str, "OrderedDict[str, Tensor]"]:
         state_dict = OrderedDict(
             {k.replace("model.", ""): v for k, v in state_dict.items()}
         )
+    elif "vits16" in checkpoint["hyper_parameters"]["backbone"]:
```

Comment:
the state dict of `self.model.encoder` is named `model.{module}`. This naming is created in the conversion script.

Reply:
This seems very hyper-specific but I don't know enough to offer an alternative